waku/standards/core/64/network.md

```diff
@@ -141,8 +141,8 @@ See the section on [autosharding](#autosharding) for more.
 
 ## RLN rate-limiting
 
-The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md) proofs
-to ensure that a pre-agreed rate limit is not exceeded by any publisher.
+The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md), extended with [RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md) proofs
+to ensure that a pre-agreed rate limit is not exceeded by any publisher. The addition of RLNv2 allows to rate-limit with more granularity, `x` amount of messages every `y` seconds.
```

Comment:
```suggestion
to ensure that a pre-agreed rate limit is not exceeded by any publisher.
The addition of RLN-V2 allows rate-limiting with more granularity, `x` amount of messages every `y` seconds.
```

As for `RLN-V2` vs `RLNv2` vs `RLNV2` (here and elsewhere) I have no particular preference as long as we use it consistently

Reply:
fix in fd45bb7495c9123dd2f00becf30643db5660e6e3 using RLN-V2